README.md

```diff
@@ -1,26 +1,7 @@
-# python-package-template
-Cookiecutter template for creating a new Python Package
+## Configuring Dev Environment
 
-# Usage
-Install cookiecutter:
+Run `make init` to configure dev environment
 
-`pip install -U cookiecutter`
+## Tests
 
-Run cookiecutter:
-
-`cookiecutter -f https://github.com/imranq2/python-package-template.git -o ../`
-
-This will ask you for the parameters.
-
-Alternatively, create a cookiecutter.json
-{
-    "directory_name": "Hello",
-    "package_name": "Howdy",
-    "author": "Julie",
-    "author_email": "foo@email.com",
-    "package_description": "description",
-    "package_github_url": "url",
-
-}
-
-After generation is complete, run `make init` to set up your environment.
```

Comment:
this doc for cookiecutter stuff has been removed, is that ok and do we have this somewhere else then??

Reply:
It is available at https://github.com/imranq2/python-package-template, but I'm glad you asked because I hadn't yet confirmed it existed elsewhere.